airflow/Dockerfile

```diff
@@ -49,15 +49,16 @@ RUN set -ex \
     && update-locale LANG=en_US.UTF-8 LC_ALL=en_US.UTF-8 \
     && useradd -ms /bin/bash -d ${AIRFLOW_HOME} airflow \
     && python -m pip install -U pip \
+    && pip install setuptools \
     && pip install Cython \
     && pip install pytz \
     && pip install pyOpenSSL \
     && pip install ndg-httpsclient \
     && pip install pyasn1 \
     && pip install psycopg2 \
     && pip install docker-py>=1.6.0 \
-    && pip install ast \
     && pip install apache-airflow[crypto,celery,postgres,hive,hdfs,jdbc]==$AIRFLOW_VERSION \
+    && pip install python-openstackclient \
```

Comment:
is it possible for you to pin the version of python-openstackclient as this is quite a fast moving target?